### DISCUSSION TITLE
Malicious code

### DISCUSSION THREAD

**Discussion by MarinaNad** [community/community#14064] [Other Feature Feedback, Questions, & Ideas]
Problem with one of the plugins by https://status301.net. I don't know if this developer's plugin was intentionally injected with malicious code or if there is a vulnerability in his plugin. The virus harmed my site through the plugin of this developer. The site is dedicated to art. I know that some developers are competing here, how else to harm the Russians.

**Comment by queenofcorgis** [comment]
Hi @MarinaNad 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms.  I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
